README.md

```diff
@@ -1 +1,60 @@
-# capstone-project-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Web Development Course Capstone Project
+
+> This is my capstone project website showcasing my skills and mastery of HTML, CSS, JAVASCRIPT and responsive design.
+
+![Capstone project](img/codeniversity_desktop2.PNG)
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## Live Demo 🔗
```

Comment:
- [ ] OPTIONAL
Your Pull Request does not include a video presentation showcasing your project, as required. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The video should:
- contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- contain a demo of the project, showing the pages in different screen sizes.
- contain a highlight about a piece of the code the student finds interesting or is proud of making.